webpack/assets/javascripts/react_app/components/ConfigReports/Welcome.js

```diff
@@ -1,18 +1,34 @@
 import React from 'react';
+import { FormattedMessage } from 'react-intl';
 import { translate as __ } from '../../common/I18n';
 import EmptyState from '../common/EmptyState';
 import { getManualURL, getWikiURL } from '../../common/helpers';
 
 export const WelcomeConfigReports = () => {
-  const content = __(`If you wish to configure Puppet to forward its reports to Foreman, 
-  please follow <a href=${getManualURL(
-    '3.5.4PuppetReports'
-  )}>setting up reporting</a> and
-  <a href=${getWikiURL('Mail_Notifications')}>e-mail reporting</a>`);
   const description = (
     <>
       {__("You don't seem to have any reports.")}
-      <div dangerouslySetInnerHTML={{ __html: content }} />
+
+      <br />
+      <FormattedMessage
+        id="report-manual"
+        defaultMessage={__(
+          'If you wish to configure Puppet to forward its reports to Foreman, please follow {settingUpPuppetReports} and {emailReporting}'
+        )}
+        values={{
+          settingUpPuppetReports: (
+            <a href={getManualURL('3.5.4PuppetReports')}>
+              {__('setting up reporting')}
+            </a>
+          ),
+          emailReporting: (
+            <a href={getWikiURL('Mail_Notifications')}>
+              {__('e-mail reporting')}
+            </a>
+          ),
```

Comment:
@ezr-ondrej I'm not sure it was ready to merge. The PRs mentioned in https://github.com/theforeman/foreman/pull/8930#discussion_r762904260 were merged so I assumed this was waiting for an update of this PR.

Is there at least an issue tracking the remaining issues with this? This is still very hard for translators to get right.

Reply:
And then reverted, I did not notice that comment and assumed it was resolved, my bad, I've at least opened a https://projects.theforeman.org/issues/34392, but it is blocked by https://projects.theforeman.org/issues/30665 which I've reopened as it was reverted and I even tried to reopen and fix the issues it's causing, but failed teribly (I do not know enough about webpack and what it does to properly fix it).

So I belive this is better off merged, because I do not see us upgrading react-intl any time soon (someone needs to find the time to figure out what is really going on)